fixtures/src/components/controlled/select.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Select, Option } from 'carbon-react/lib/__experimental__/components/select';
+import { LogConsumer } from '../log';
+
+const ControlledSelect = () => {
+  const [selectedValue, setSelectedValue] = useState();
+  const [selectedValueExampleTwo, setSelectedValueExampleTwo] = useState('2');
+  return (
+    <LogConsumer>
+      {(log) => {
+        const onChange = setter => (e) => {
+          log(e, { method: 'onChange' });
+          setter(e.target.name);
```

Comment:
Just wondering, shouldn't this be `setter(e.target.value)` ?

Reply:
I think this actually should be `setter(e.target.value.map(o => o.optionValue))` after your https://github.com/Sage/carbon/pull/2422 PR gets merged

Reply:
@mkrds: Ah yes, very good point :thumbsup: There's an example of that mapping here:

* https://github.com/Sage/carbon/blob/625a1b9/src/__experimental__/components/select/select.stories.js#L40

@edleeks87: Thanks for fixing this in fe2a03d :white_check_mark:

Reply:
I've gone with `setter(e.target.value[0].optionValue);` as currently the fixture is only implementing a s`SingleSelect` 
If we in turn implement the `MultiSelect` variant then `setter(e.target.value.map(o => o.optionValue)` will be needed